options.go

```diff
@@ -143,7 +143,7 @@ type Options struct {
 	readOnly bool
 
 	// Disable set-lib on connect. Default is false.
-	DisableIndentity bool
+	DisableIdentity bool
```

Comment:
Renaming this will be a breaking change for everyone setting it now.

Reply:
Thank you!